.github/workflows/on_push.yml

```diff
@@ -105,3 +105,41 @@ jobs:
         --container api-container \
         -- /opt/app/bin/liquid_voting \
         eval "LiquidVoting.Release.migrate"
+
+  smoke_tests:
+    name: Smoke Tests
+    needs: [deploy]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        repository: liquidvotingio/deployment
+        token: ${{ secrets.DEPLOYMENT_PAT }}
+        path: deployment
+    - name: Run k6 smoke tests
+      uses: k6io/action@v0.1
+      env:
+        TEST_API_AUTH_KEY: ${{ secrets.TEST_API_AUTH_KEY }}
+      with:
+        filename: deployment/smoke_tests/smoke_tests.js
+    - name: Login to gcloud
+      uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+      with:
+        version: '270.0.0'
+        service_account_email: ${{ secrets.GKE_EMAIL }}
+        service_account_key: ${{ secrets.GKE_KEY }}
+    - name: Teardown smoke test data
+      env:
+        GKE_CLUSTER: ${{ secrets.GKE_CLUSTER }}
+        GKE_ZONE: ${{ secrets.GKE_ZONE }}
+        GKE_PROJECT: ${{ secrets.GKE_PROJECT }}
+      run: |
+        gcloud container clusters get-credentials $GKE_CLUSTER --zone $GKE_ZONE --project $GKE_PROJECT
+        pod=$(kubectl get pods -l app=api  -o json | jq -r '.items[].metadata.name' | head -1)
+        kubectl wait --timeout=120s --for=condition=Ready pod/$pod
+        kubectl exec -ti pod/$pod \
+        --container api-container \
+        -- /opt/app/bin/liquid_voting \
+        eval "LiquidVoting.Release.smoke_test_teardown"
+      
```

Comment:
Missing newline 😉

Reply:
Interestingly not showing here but only in the files changed tab.

Reply:
There is a newline at EOF. It was the last PR, but before I requested your review ;)